sdk/spring-sdk/src/main/scala/kalix/springsdk/KalixConfiguration.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk
+
+import kalix.javasdk.action.ActionCreationContext
+import kalix.javasdk.valueentity.ValueEntityContext
+import kalix.javasdk.view.ViewCreationContext
+import kalix.springsdk.impl.KalixServer
+import kalix.springsdk.impl.KalixServer.ActionCreationContextFactoryBean
+import kalix.springsdk.impl.KalixServer.ValueEntityContextFactoryBean
+import kalix.springsdk.impl.KalixServer.ViewCreationContextFactoryBean
+import org.springframework.context.ApplicationContext
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class KalixConfiguration(applicationContext: ApplicationContext) {
```

Comment:
This KalixConfiguration will 'teach' Spring which bits it needs to include in the application context. 

The first part is the `KalixServer` itself. Next to it, we need  ThreadLocal backed FactoryBeans to provide the different creation contexts when needed. That's the hack!